src/sentry/web/frontend/base.py

```diff
@@ -419,11 +434,12 @@ def needs_sso(self, request: Request, organization):
         return False
 
     def convert_args(self, request: Request, organization_slug=None, *args, **kwargs):
-        active_organization = self.get_active_organization(
-            request=request, organization_slug=organization_slug
-        )
+        organization: Optional[Organization] = None
+        if self.active_organization:
+            for org in Organization.objects.filter(id=self.active_organization.id):
+                organization = org
```

Comment:
Any reason to not use `.objects.get(id=self.active_organization.id)` here?

Reply:
Just a lack of familiarity with django's orm, I'll take a closer look again.

Reply:
Is there a `get` like method but without the exception?  Essentially I want a `get_or_none` but I don't see that in any documentation.